src/elementary/NuclideID.hpp

```diff
@@ -95,7 +95,7 @@ namespace elementary {
      */
     int hash() const noexcept {
 
-      return this->za() * 100 + this->level_.number();
+      return this->za() * 1000 + this->level_.number();

```

Comment:
Why did this change to `1000`? Should it have been this all along and the previous reviewer (aka @jlconlin) missed it?

Reply:
I use the maximum possible value for an unsigned char to represent the continuum (which is larger than 100), so I need to multiple the ZA by 1000 instead of 100. Previously, the LevelNumber class (now Level) essentially only stored levels 0 to 99 which is more than enough. From a user point of view, the behaviour has not changed though.